UI/cmake/ui-qt.cmake

```diff
@@ -1,4 +1,10 @@
-find_qt(COMPONENTS Widgets Network Svg Xml COMPONENTS_LINUX Gui)
+# cmake-format: off
+find_package(Qt6 REQUIRED Widgets Network Svg Xml)
```

Comment:
As with other places I just prefer using the shorthand, but have no hard objection to adding it. In any case I'd want this to be consistent across the codebase though.

Reply:
LMK if you have come to a decision regarding the short-hand form, as I would need to update it throughout the code-base to ensure consistency then. Right now it should use the shortened form throughout.

Reply:
Let's just use the short form since that's already in this PR. Will keep the churn down and reduce chance of mistakes slipping in. We can always change it later if we find a reason to do so.